viz.yaml

```diff
@@ -0,0 +1,68 @@
+vizlab: "0.3.6"
+info:
+  id: gage-conditions
+  name: USGS Gage Conditions
+  date: 2018-06-11
+  publish-date: 2019-06-25
+  path: /gage-conditions
+  analytics-id: # WE WILL NEED TO FILL THIS IN AT SOME POINT
+  description: Map of all USGS gages and their conditions.
+  required-packages:
+    vizlab:
+      repo: github
+      version: 0.3.6
+      name: USGS-VIZLAB/vizlab
+      ref: v0.3.6
+    dataRetrieval:
+      repo: CRAN
+      version: 2.7.4
+    dplyr:
+      repo: CRAN
+      version: 0.7.4
+  contributors:
+    -
+      name: Lindsay Carr
+      email: lcarr@usgs.gov
+      affiliation: U.S. Geological Survey
+    -
+      name: Jordan Read
+      email: jread@usgs.gov
+      affiliation: U.S. Geological Survey
+parameter:
+  - id: plot_date
+    date: "2018-05-01"
+fetch:
+  - id: dv_sites
+    location: cache/dv_sites.json
+    reader: json
+    fetcher: dv_sites
+    scripts: [scripts/fetch/dv_sites.R]
+    depends:
+      date: plot_date
+  - id: site_stats # took about 20 min
```

Comment:
this is an expensive fetch, and I think we'll want to do something so it doesn't trigger everytime we change `plot_date`. This fetches all of the data for a given year (can't remember if it is calendar year or water year), so I think we might want to have another global called "year" or "water_year", and then "plot_date" is the date of that year (maybe "05-01" here?). `site_stats` could then depend on `year` _instead_ of `plot_date`, and save us a lot of time as we bounce around to different dates of the year. Also helps us if we add a time dimension to this viz.

Reply:
A little confused on this one. I don't currently have site_stats depending on the date, but thinking about it now I realize that it should depend on the date. It currently is only doing:

```
readNWISstat(siteNumbers = get_sites,
                   parameterCd = "00060", 
                   statReportType="daily",
                   statType=paste0("P", stat_types)
```

Reply:
Actually, this currently takes 20 min to get stats for all time. I should probably include the year to timebox somewhat.

Reply:
I'm rethinking this - 20 min for all time doesn't seem so bad. Then you wouldn't need to run this ever again.